packages/output/src/browser/output-widget.tsx

```diff
@@ -93,13 +94,16 @@ export class OutputWidget extends BaseWidget implements StatefulWidget {
         this.onStateChangedEmitter.fire(this._state);
     }
 
-    protected async refreshEditorWidget(): Promise<void> {
+    protected async refreshEditorWidget({ preserveFocus }: { preserveFocus: boolean } = { preserveFocus: false }): Promise<void> {
```

Comment:
why not have boolean argument instead of object with single boolean property?

Reply:
We can easily extend it without breaking the API in the future; it's pretty common that we use an object instead of a single property. I can make the change, though. I am OK with both.

Reply:
i'm ok with both if we think it might expand in the future. consider having a named interface so it is more readable.